### PR TITLE
build: add Concourse pipeline for container images 

### PR DESCRIPTION
**Pull request by basti1302**

### CODE REVIEW

**Comment by basti1302** [comment]
See also https://github.ibm.com/instana/delivery-infra/pull/94

File: packages/serverless/ci/pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+        alert_type: success
+        message: |
+              :white_check_mark: The Instana Node.js AWS Lambda layer for package version ((.:package-version)) and the corresponding container image `icr.io/instana/aws-lambda-nodejs` have been released.
+    on_failure: &slack-notify-failure-lambda
+      put: slack-alert
+      params:
+        channel: '#team-node'
+        alert_type: failed
+        message: |
+              :x: Building/pushing a new version of the container image `instana/aws-lambda-nodejs` has failed.
```
Comment (kirrg001) [inline packages/serverless/ci/pipeline.yaml:330]: Want to add `icr.io/` here too?

**Review by kirrg001 — APPROVED** [review]
I guess we need to update our docs with the new container images references
e.g. https://www.instana.com/docs/ecosystem/aws-fargate/

**Comment by basti1302** [comment]
> I guess we need to update our docs with the new container images references e.g. https://www.instana.com/docs/ecosystem/aws-fargate/

Absolutely, but only after we have published the images there. I have that on my list. 👍

File: packages/serverless/ci/pipeline.yaml (hunk 1 above)
Comment (basti1302) [reply packages/serverless/ci/pipeline.yaml]: Yep, fixed that message, thanks for spotting this.

**Pull request merged**